js/script.js

```diff
@@ -1,51 +1,67 @@
 'use strict';
 
+// define html elements
 var output = document.getElementById('output');
-
 var btn = document.getElementById('btn');
 
-var temp;
+// variables
 var celsius;
 var fahrenheit;
 
-var msg = function(temp) {
-  if (temp <= -28) {
-    output.innerHTML += 'It\'s freezing. Stay at home!';
-  } else  if (temp <= 12) {
-    output.innerHTML += 'It\'s cold. Take something with long sleeves.';
-  } else if (temp >= 28) {
-    output.innerHTML += 'Don\'t forget about something to drink.<br>'+
+// get celsius and return fahrenheit
+var getFahrenheit = function(celsius) {
+  fahrenheit = Math.floor((celsius * 1.8) + 32);
+  return fahrenheit;
+}
+
+// get celsius and return message
+var getMsg = function(celsius) {
+  if (celsius <= -28) {
+    return 'It\'s freezing. Stay at home!';
+  } else if (celsius <= 12) {
+    return 'It\'s cold. Take something with long sleeves.';
+  } else if (celsius >= 28) {
+    return 'Don\'t forget about something to drink.<br>'+
     ' It\'s hot outside.';
   } else {
```

Comment:
last else is unnecessary, because you are returning string in every if statement. return will stop the function